src/main/java/frc/robot/commands/JsonUpdateCommand.java

```diff
@@ -0,0 +1,59 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.CoordsOutputSubsystem;
+
+public class JsonUpdateCommand extends Command{
+    
+
+    private CoordsOutputSubsystem s_CoordsOutputSubsystem;
+
+    private Timer timer;
+
+    private double timeOffset;
+
+    private int id = 0;
+
+
+    public JsonUpdateCommand() {
+
+        s_CoordsOutputSubsystem = CoordsOutputSubsystem.getInstance();
+
+        timer = new Timer();
+
+    }
+
+    @Override
+    public void initialize() {
+        timeOffset = Timer.getFPGATimestamp();
+    }
+
+    @Override
+    public void execute() {
+        
+        if (timer.hasElapsed(0.5 + timeOffset)) {
+
+            Pose2d botPose = s_CoordsOutputSubsystem.s_Swerve.getPose();
```

Comment:
You may want to also output the pose from getEstimatedPosition(), since it uses the swerveEstimator.

Reply:
done